rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec

```diff
@@ -290,9 +290,19 @@ fi
 
 # Need to restart to take schema changes into account
 echo -n "INFO: Restarting rudder-slapd..."
-/sbin/service rudder-slapd restart >/dev/null 2>&1
+/sbin/service rudder-slapd force-reload >/dev/null 2>&1
```

Comment:
Shouldn't we keep restart so that if it was not started before, it is started now ?